src/components/app/App.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import PaymentsDetails from '../payments-details/paymentsDetails';
+import {BrowserRouter, Switch} from 'react-router-dom';
+import MainContent from '../main_content/mainContent'
+
+
+import {Route} from 'react-router-dom';
+
+import './app.css';
+
+
+export default class App extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            users: [],
+            spendings: [],
+
+            sumOfGroupSpent : 0,
+        }
+        ;
+
+        this.onAdd = this.onAdd.bind(this);
```

Comment:
as we've discussed in the previous cycle of code review - the `.bind` isn't needed when you use the arrow function assignment
please refer to this comment for further examples
https://github.com/webdevbyjoss/RoadTripSpendings/pull/2#discussion_r498377603

and lest get rid of `.bind()` calls all across the codebase